cockatrice/src/pictureloader.cpp

```diff
@@ -0,0 +1,462 @@
+#include "pictureloader.h"
+#include "carddatabase.h"
+#include "main.h"
+#include "settingscache.h"
+#include "thememanager.h"
+
+#include <QApplication>
+#include <QCryptographicHash>
+#include <QDebug>
+#include <QDir>
+#include <QFile>
+#include <QImageReader>
+#include <QNetworkAccessManager>
+#include <QNetworkReply>
+#include <QNetworkRequest>
+#include <QPainter>
+#include <QPixmapCache>
+#include <QSet>
+#include <QSvgRenderer>
+#include <QThread>
+#include <QUrl>
+
+class PictureToLoad::EnabledAndKeyCompareFunctor {
```

Comment:
I know it's C++ convention to call this Functor, but can we please not? It's extremely confusing when you're used to using actual functors (from category theory).

`EnabledAndKeyCompareFunction` would be fine by me..